### PR TITLE
Bug fix in logs

### PR DESCRIPTION
**Pull request by ChetanGN**

### CODE REVIEW

File: woo-razorpay.php [hunk 1]
```diff
@@ -1342,7 +1342,7 @@ public function updateOrder(& $order, $success, $errorMessage, $razorpayPaymentI
 
             $orderId = $order->get_order_number();
 
-            rzpLogInfo("updateOrder orderId: $orderId , errorMessage: $errorMessage, razorpayPaymentId: $razorpayPaymentId , success: $success");
```
Comment (abdulwahidsharief) [inline woo-razorpay.php:1345]: @ChetanGN why $errorMessage is removed ?

Comment (ChetanGN) [reply woo-razorpay.php:1345]: It was leading to confusion as merchant logs have this error message even if the payment was successful. So we don't need to print this on logs. @abdulwahidsharief

**Review by poojarazor — APPROVED** [review]
LGTM

**Review by angbpy — APPROVED** [review]
LGTM

**Pull request merged**